include/zephyr/dt-bindings/gpio/nxp-imx-gpio.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Vestas Wind Systems A/S
```

Comment:
Very kind, but Vestas does not hold copyright on this file and the year should say 2025.